docs/sysadmin-basics/cluster-config.txt

```diff
@@ -831,8 +832,9 @@ The master supports the following configuration settings:
    -  ``auto``: Automatically detects the slot type. The agent will detect if there are GPUs. If
       there are GPUs, it maps each GPU to one slot. Otherwise, it maps all the CPUs to a slot.
    -  ``none``: The agent will not create any slots for detected devices.
-   -  ``gpu``: The agent will map each detected GPU to a slot.
+   -  ``gpu``: The agent will map each detected NVIDIA GPU to a slot.
```

Comment:
it's _almost_ worth renaming `gpu` to be `nvidia`, but that sounds like an obnoxious migration...